src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -2697,7 +2697,7 @@ bool ColorControlServer::moveColorTempCommand(app::CommandHandler * commandObj,
     {
         colorTemperatureMinimum = tempPhysicalMin;
     }
-    if (colorTemperatureMaximum > tempPhysicalMax)
+    if ((colorTemperatureMaximum == 0) || (colorTemperatureMaximum > tempPhysicalMax))
```

Comment:
The cited issue talks about the min temperature too, no?  Why is this PR only changing the maximum?

Reply:
The min temperature is already covered by the existing if condition  as tempPhysicalMin cannot be lower than 0 anyway
```
if (colorTemperatureMinimum < tempPhysicalMin)
{
        colorTemperatureMinimum = tempPhysicalMin;
}
```

The issue only applied to `colorTemperatureMaximum` since the check was greater than only ence not catching the 0 condition.

Reply:
It might be worth noting in a code comment that the spec requirement around `colorTemperatureMinimum == 0` is handled by the fact that `tempPhysicalMin >= 0`.